google/datalab/bigquery/_sampling.py

```diff
@@ -111,3 +111,25 @@ def _random_sampling(sql):
         sql = '%s LIMIT %d' % (sql, count)
       return sql
     return _random_sampling
+
+  @staticmethod
+  def _auto(method, fields, count, percent, key_field, ascending):
+    """Construct a Sorting object according to the provided sampling technique, provided all
```

Comment:
Change "Sorting object" to either "sampling function" or "sampling strategy" to be consistent with how the rest of this file refers to the type of value returned from this method.